src/Microsoft.AspNet.Http.Extensions/Encoders/EncoderServices.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Framework.ConfigurationModel;
+using Microsoft.Framework.DependencyInjection;
+
+namespace Microsoft.AspNet.WebUtilities.Encoders
+{
+    public static class EncoderServices
+    {
+        public static IEnumerable<IServiceDescriptor> GetDefaultServices(IConfiguration configuration = null)
```

Comment:
Would the parameterless overload just call the parameterized overload passing null?